config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] It would be better if our routes matches up with the given wireframe, currently, it is not matching the wireframe. Here is a screenshot to show `our routes`, `required routes` and `wireframe`
Wireframe
![Screenshot from 2023-07-19 20-31-43](https://github.com/mercymugambi/blog-app/assets/46388113/06f9c538-e530-48f4-ba1d-dab6826205c4)
Our route
![Screenshot from 2023-07-19 20-31-08](https://github.com/mercymugambi/blog-app/assets/46388113/39bfee13-1ce1-48b4-89e6-fee30913cac0)
Required route
![Screenshot from 2023-07-19 20-30-30](https://github.com/mercymugambi/blog-app/assets/46388113/31b975e1-fa33-4336-8689-490f72d72a12)
Here is how we can fix it
```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
Nesting the resources